File: backend/schemas/pizzaSchemas.py
from pydantic import BaseModel, EmailStr
from typing import Optional

class PizzaBase(BaseModel):
    nome: str
    ingredientes: str
    preco: float
    id: str
    imagem_url: Optional[str] = None
    data_pedido: Optional[str] = None
    cliente_id: Optional[str] = None

File: backend/schemas/pizzaSchemas.js
const Joi = require('joi');

// Schema for PizzaBase
const PizzaBaseSchema = Joi.object({
  nome: Joi.string().required(),
  ingredientes: Joi.string().required(),
  preco: Joi.number().required(),
  id: Joi.string().required(),
  imagem_url: Joi.string().uri().optional(),
  data_pedido: Joi.string().optional(),
  cliente_id: Joi.string().optional(),
});

module.exports = {
  PizzaBaseSchema,
};